src/hypofuzz/hypofuzz.py

```diff
@@ -483,6 +493,14 @@ def callback(test_case: dict) -> None:
             # because it doesn't find a pytest item context (I didn't look further
             # than that).
             test_case["property"] = self.nodeid
+            # run_start is relative to StateForActualGivenExecution, which we
+            # re-use per FuzzProcess. Overwrite with the current timestamp, so that
+            # the e.g. dashboard can reliably use this for sorting observations.
+            test_case["run_start"] = time.time()
```

Comment:
code is fine, but in the comment - this is an _unreliable_ (but good enough) way to sort observations.